cmd/multiclusterapp.go

```diff
@@ -120,11 +135,22 @@ func MultiClusterAppCommand() cli.Command {
 						Usage: "Access type of the members. Specify only one value, and it applies to all members defined by --member. Valid options are 'owner', 'member' and 'read-only'",
 						Value: "owner",
 					},
-					cli.IntFlag{
-						Name:  "timeout",
-						Usage: "Time in seconds to wait until the app is in a ready state",
-						Value: 60,
+					cli.StringFlag{
+						Name:  argUpgradeStrategy,
```

Comment:
I don't understand why we have upgrade strategy in install command. Can you explain a little?

Reply:
Upgrade strategy is an attribute of multiclusterapp. It's not dedicated for a single upgrade.